shell/components/ClusterIconMenu.vue

```diff
@@ -106,10 +107,12 @@ export default {
     <i
       v-if="!routeCombo && cluster.pinned"
       class="icon icon-pin cluster-pin-icon"
+      :alt="t('nav.ariaLabel.pinCluster', { cluster: 'Local' })"
```

Comment:
this is in a different if block to above, and could show either local or other clusters, so should have the actual cluster name

Reply:
You are right... I confused the usage with other "cluster" components. This one may be used by a cluster different than local. Fixed